serieMotion/caracteristicas.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+from funciones import caracteristicaValida
+
+class Caracteristicas:
+    
+
+    def __init__(self,accion,terror,romance,comedia,efectos_especiales, valoracion):
+        
+        #Todas las características serán valores reales del [0.0 - 5.0]
+        self.accion = caracteristicaValida(accion)
+        self.terror = caracteristicaValida(terror)
+        self.romance = caracteristicaValida(romance)
+        self.efectos_especiales = caracteristicaValida(efectos_especiales)
+        self.valoracion = caracteristicaValida(valoracion)
+        
+        
+    ####  Gets  ####
+        
+    def getAccion(self):
+        return self.accion 
+    
+    def getTerror(self):
+        return self.terror 
+    
+    def getRomance(self):
+        return self.romance 
+    
+    def getEfectos_especiales(self):
+        return self.efectos_especiales
+    
+    def getValoracion(self):
+        return self.valoracion
+    
+    
+    ####  Sets  ####
+    
+    def setAccion(self, accion):
```

Comment:
¿Donde dice que tiene que haber getters y setters para cada caracerística?

Reply:
Y donde no? se supone que lo basico de  una estructura de clases es que te permita modificar los datos, no lo veo un problema

Reply:
De hecho hice gets para mi objetivo 2  sobre el trabajo de @NachoCarher y no me dijiste nada

Reply:
¿Entendéis que el desarrollo del código tiene que ser según un issue, HU y siguiendo la lógica de negocio? Este código está cambiado en 6f167a01bd72b8dce20f716b22f9e40a11398a65 junto con 3 ficheros más. No hay ningún issue ni por supuesto ninguna HU. Por lo tanto, es código sin ninguna relación con lo que se pretende hacer en el proyecto. Eso es válido para el resto del fichero, pero en este caso está mucho más claro.
Por otro lado, os rogaría que tratárais de interpretar lo que se os dice en la revisión y no discutirlo, para aprovechar mejor el tiempo de todos.

Reply:
Ahí ha pasado algo raro, porque como dicho en el mensaje de commit solamente habia añadido poetry y no he metido nada de código, mira este commit https://github.com/Parka015/SerieMotion-IV/commit/889ca711878cf9129a566cd319d7561515b29464 creo que ha sido por renombrar la carpeta de los fuentes, de cara a poetry, cque me lo ha contado como si fuesen archivos nuevos cuando no es el caso.
Eso son problemas de no tener experiencia en Github pero lo he seguido a rajatabla como se ve en el commit que te he puesto ( https://github.com/Parka015/SerieMotion-IV/commit/889ca711878cf9129a566cd319d7561515b29464 ) ahí si es donde hice el código

Reply:
Todos los cambios de código tienen que seguir un issue, también los cambios de nombre. En todo caso, el problema principal sigue ahí.